web_widget_tolocalestring/__openerp__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# (c) 2016 Tony Galmiche / InfoSaône
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Web Widget toLocaleString",
+    "summary": """
```

Comment:
remove or " 'tolocalestring' widget improves numbers formatting or allows to not display the 0 values.